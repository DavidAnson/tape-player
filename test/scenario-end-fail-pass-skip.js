// @ts-check

"use strict";

const tape = require("tape");
require("../index.js");

tape("end", (test) => {
  test.end("end error");
});

tape("pass fail", (test) => {
  test.pass("pass message");
  test.fail("fail message");
  test.end();
});

tape("skip", (test) => {
  test.plan(3);
  test.pass("before message");
  test.skip("skip message");
  test.pass("after message");
});
