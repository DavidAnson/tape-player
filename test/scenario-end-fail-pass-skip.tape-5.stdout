end
  scenario-end-fail-pass-skip.js:9:8
  Message:  end error
  Operator: error
pass fail
  scenario-end-fail-pass-skip.js:14:8
  Message:  fail message
  Operator: fail
skip

Tests:    3
Asserts:  6
Failures: 2
